src/protocol/logpoint.ts

```diff
@@ -536,9 +538,15 @@ export async function setExceptionLogpoint(logGroupId: string) {
   try {
     await analysisManager.runAnalysis(params, handler);
   } catch (e: any) {
+    removeLogpoint(logGroupId);
+    let msg;
     if (e.code === ProtocolError.TooManyPoints) {
-      store.dispatch(onConsoleOverflow());
+      msg = "There are too many exceptions. Please focus and try again.";
```

Comment:
:hand: Might be worth making this message a bit longer, like "Please focus to a smaller time range and try again".